### PR TITLE
New Port: Formula 1 Playdate

### PR DESCRIPTION
**Pull request by joyrider3774**
# New Port for Formula 1 Playdate

## Game Information
- **Title**: Formula 1 Playdate
- **URL**: https://github.com/joyrider3774/formula1_playdate (also requires api runtime https://github.com/joyrider3774/Playdate_Api_SDL2 )

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

**Review by Cebion — APPROVED** [review]

File: ports/formula_1_playdate/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Notes
+<br/>
+
+Thanks to [Willems Davy aka joyrider3774](https://github.com/joyrider3774) for this game and making available for free. Also thanks to Willems Davy aka Joyrider3774 for the packaging for portmaster.
```
Comment (kloptops) [inline ports/formula_1_playdate/README.md:4]: LOL

**Review by kloptops — APPROVED** [review]
👍

File: ports/formula_1_playdate/README.md (hunk 1 above)
Comment (joyrider3774) [reply ports/formula_1_playdate/README.md:4]: yeah was not sure what todo there lol :)

**Review by christopher-roelofs — COMMENTED** [review]
gtg

**Pull request merged**